r/R/parquet.R

```diff
@@ -55,36 +55,43 @@ read_parquet <- function(file,
 #' @param sink an [arrow::io::OutputStream][OutputStream] or a string which is interpreted as a file path
 #' @param chunk_size chunk size in number of rows. If NULL, the total number of rows is used.
 #'
-#' @param version parquet version, "1.0" or "2.0".
-#' @param compression compression algorithm. No compression by default.
-#' @param compression_level compression level.
-#' @param use_dictionary Specify if we should use dictionary encoding.
-#' @param write_statistics Specify if we should write statistics
-#' @param data_page_size Set a target threshhold for the approximate encoded size of data
-#'        pages within a column chunk. If omitted, the default data page size (1Mb) is used.
+#' @param version parquet version, "1.0" or "2.0". Default "1.0"
+#' @param compression compression algorithm. Default "snappy"
```

Comment:
It would be good to list the valid `compression` values somewhere. Perhaps in the Codec docs, and you can link to them from here. Here are the possible values (upper or lower case are supported), though availability of course depends on the build: https://github.com/apache/arrow/blob/cf9df1498a214dc72d52010c3d2a80e92fcc6e33/r/R/enums.R#L79

It would also be good to indicate how to turn off compression since "snappy" is now default. (I believe it would be `compression = "uncompressed"`)

Also should clarify that snappy is used if it is available, which it should be most everywhere but technically it's not guaranteed, else uncompressed is default.